spec/models/request_form/subject_id_spec.rb

```diff
@@ -3,5 +3,32 @@
 RSpec.describe RequestForm::SubjectId, type: :model do
   it_behaves_like "question when requester is not a solicitor"
   it_behaves_like "file upload", :subject_photo
-  it_behaves_like "file upload", :subject_proof_of_address
+
+  describe "#valid_file_type" do
```

Comment:
What about if the file type is not valid?

Reply:
added test for invalid file type